templates/static/images/icons/hamburger-black.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M24 6h-24v-4h24v4zm0 4h-24v4h24v-4zm0 8h-24v4h24v-4z"/></svg>
```

Comment:
Thanks @gtvj - Is this irrelevant here, as this icon is only shown via CSS with `background-image`? When CSS is turned off, the button becomes a button with text instead of an icon.

Reply:
Good point.